src/GlobalStylesheets.js

```diff
@@ -87,6 +85,7 @@ var GlobalStylesheets = {
       var stylesheet = {
         id: GlobalStylesheets.injection.getStylesheetId(key),
         style: styleObj,
+        displayName: displayName,
```

Comment:
Would be nice if this was the parent component's displayName.  (e.g. `Block` is not as meaningful as `Avatar`)  The loader uses the filename which usually achieves that, not sure how to do it here.